pdfannots/printer/markdown.py

```diff
@@ -90,12 +90,14 @@ def __init__(
         remove_hyphens: bool = True,            # Whether to remove hyphens across a line break
         use_page_labels: bool = True,           # Whether to use page labels
         wrap_column: typ.Optional[int] = None,  # Column at which output is word-wrapped
+        page_number_offset: typ.Optional[int] = 0,  # Page number offset
```

Comment:
nit: please keep named arguments sorted, i.e. move to line 89

Reply:
Done.